src/plugins/navigation/public/top_nav_menu/top_nav_menu.test.tsx

```diff
@@ -66,28 +65,24 @@ describe('TopNavMenu', () => {
 
   it('Should render nothing when no config is provided', () => {
     const component = shallowWithIntl(<TopNavMenu appName={'test'} />);
-    expect(component.find(WRAPPER_SELECTOR).length).toBe(0);
```

Comment:
So, it looks like the primary purpose of that class was for unit testing. What do we lose from the integrity of these unit tests by removing the wrapper assertions?

Reply:
@andreymyssak Can you answer this question? Then I can approve.

Reply:
@joshuarrrr I think we lose nothing in terms of the integrity of the tests. There are only two elements in the component (`SearchBar` and `TopNav`) that are meaningful when testing. The `WRAPPER_SELECTOR` test is not meaningful because it is just a wrapper for the previously mentioned elements, which are tested anyway, whether there is a `WRAPPER_SELECTOR` or not, so the `WRAPPER_SELECTOR` test is pretty useless.